src/diff_tests/diff_tests.cpp

```diff
@@ -21,16 +21,25 @@ string exec_command(const char* cmd) {
     return result;
 }
 
-string extractData(const string str) {
+string extractData(const string str, string type) {
+
     stringstream iss(str);
     string line;
+    std::stringstream read_write_svfg_ss;
     while(getline(iss, line)) {
-        if (line.find("AvgTopLvlPtsSize") == 0) {
-            // cout << line << endl;
-            return line;
+        string temp = line; 
+        if(type == "read_write_svfg"){
+            if (temp.find("AvgTopLvlPtsSize") != std::string::npos || temp.find("TotalNode") != std::string::npos || temp.find("TotalEdge") != std::string::npos) {
```

Comment:
should also compare the following in SVFG Statistics:

AvgIndOutDeg        
AvgIndInDeg         
AvgOutDegree        
AvgInDegree
AvgWeight